langchain/src/chat_models/llama_cpp.ts

```diff
@@ -29,8 +30,14 @@ export interface LlamaCppInputs
 export interface LlamaCppCallOptions extends BaseLanguageModelCallOptions {
   /** The maximum number of tokens the response should contain. */
   maxTokens?: number;
-  /** A function called when matching the provided token array */
-  onToken?: (tokens: number[]) => void;
```

Comment:
This is not going to be backwards compatible. Instead we should mark as deprecated

Reply:
The `onToken` was included in error and was not actually applied as the eventual implementation used a raw call. I can mark it as deprecated and not used if preferred.